stylus/ui-app/nav.styl

```diff
@@ -0,0 +1,53 @@
+$navigation
+    .coz-nav
+      margin     em(24px) 0
+      padding    0
+      list-style none
+    .coz-nav-item
+        position    relative
+        z-index     10
```

Comment:
Why 10? Is there a lot of stuff that have z-index between 0 and 10 or do you increment by 10 ? Just asking, but it made me think that we absolutely need guidelines and a linter, the sooner the better.